src/main/scala/org/broadinstitute/dsde/workbench/sam/api/StandardUserInfoDirectives.scala

```diff
@@ -67,12 +68,12 @@ trait StandardUserInfoDirectives extends UserInfoDirectives with LazyLogging {
 
   private def requireUserInfoFromJwt(jwtUserInfo: JwtUserInfo, bearerToken: OAuth2BearerToken, icService: IdentityConcentratorService): Directive1[UserInfo] =
     onSuccess {
-      getUserInfoFromJwt(jwtUserInfo, bearerToken, directoryDAO, icService).unsafeToFuture()
+      getUserInfoFromJwt(jwtUserInfo, bearerToken, directoryDAO, icService, SamRequestContext(null)).unsafeToFuture()
     }
 
   private def requireUserInfoFromOIDC(oidcHeaders: OIDCHeaders): Directive1[UserInfo] =
     onSuccess {
-      getUserInfoFromOidcHeaders(directoryDAO, oidcHeaders).unsafeToFuture()
+      getUserInfoFromOidcHeaders(directoryDAO, oidcHeaders, SamRequestContext(null)).unsafeToFuture()
```

Comment:
for `requireUserInfoFromJwt` and `requireUserInfoFromOIDC`, tracing seems unnecessary since they are used to authenticate the user?

Reply:
Still part of the request, so we still care how long it takes would be my opinion. Unless it's an absolute pain to add due to how we have our akka structured.
(missed this on the first go around.)

Reply:
as discussed in standup, we'll pull this into a separate ticket/PR: https://broadworkbench.atlassian.net/browse/CA-849